File: Data Structures & Algorithms/concatenation-of-array/submission-3.py
class Solution:
    def getConcatenation(self, nums: List[int]) -> List[int]:
        n=len(nums)
        result=[0]*(2*n)
        j=0
        for i in nums:
            result[j]=i
            result[n+j]=i
            j+=1
        return result    
        

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-3.cpp
class Solution {
public:
    int longestConsecutive(vector<int>& nums) {
        unordered_set<int>mp(nums.begin(),nums.end());
        int longest=0;
        for(int i=0;i<nums.size();i++)
        {
            if(mp.find(nums[i]-1)==mp.end())
            {
                int count=1;
                int current=nums[i];

                while(mp.find(current+1)!=mp.end())
                {
                    count++;
                    current++;
                }
                longest=max(longest,count);
            }
            
        }
        
        return longest;
    }
};
